src/routes/lobby/LobbyView.vue

```diff
@@ -141,7 +142,19 @@ const opponentUsername = computed(() => gameStore.opponentUsername);
 // Methods
 async function ready() {
   readying.value = true;
-  await gameStore.requestReady();
+  try {
+    await gameStore.requestReady();
+  } catch (err) {
+    // If game has already started; navigate to GameView
+    if (err?.code === 'GAME_ALREADY_STARTED') {
```

Comment:
I'd like to pull the string `'GAME_ALREADY_STARTED'` off the existing `CustomErrorType` enum, but I think we'll get hosed by the fact that it's in commonjs and used on the backend. I could try switching that file to json but I think that breaks our ability to autocomplete it anyway...